src/index.ts

```diff
@@ -40,14 +25,14 @@ function emitLog(appName: string, data: LogData) {
 export interface LogData {
   message: string
   stackTrace?: string
-  level: LogLevel
+  level?: LogLevel
```

Comment:
I don't think this is safe – what happens at the processing end if this is missing?